data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -345,10 +345,20 @@ then
         --volume "$PWD:/assets:z" \
         --volume /etc/kubernetes:/etc/kubernetes:z \
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
+{{if .BootstrapInPlace}}
+        start --tear-down-early=false --asset-dir=/assets --required-pods=""  --assets-create-timeout=5m
```

Comment:
remind me what tear-down-early is about?

Reply:
tear-down-early will take down the control plane once required pods are running on the master nodes.
Not relevant for the bootstrap-in-place use case